loginpage/js/loginpage.js

```diff
@@ -1,34 +1,34 @@
-$(document).ready(function()
+$(document).ready(function ()
 {
```

Comment:
Opening brace should be on the same line as the declaration

*PHPCodeSnifferBear, severity NORMAL, section `js`.*